lib/roo_on_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module RooOnRails
-  VERSION = '1.4.0'.freeze
+  VERSION = '1.4.1'.freeze
```

Comment:
yeah, wasn't sure what the versioning policy was for roo_on_rails. i guess minor is more appropriate since we're effectively enabling sidekiq monitoring.